js/components/colorTag.js

```diff
@@ -0,0 +1,11 @@
+const React = require("react");
+
+class colorTag extends React.Component {
```

Comment:
Your linter maybe let you know that react components use PascalCase all the time, not camelCase. This is because a component (Like say `Button`) doesn't get mistaken for an HTML tag (Like say `button`)